torch/lib/c10d/ProcessGroupGloo.cpp

```diff
@@ -153,6 +153,28 @@ void initializeStreamsEvents(
   }
 }
 
+void initializeStreamsEvents(
```

Comment:
Please add a comment similar to the one above with the of expectation that outputs in the inner vector are located on the same device.

Reply:
I reimplemented it and removed the assumption.